dask/bag/tests/test_bag.py

```diff
@@ -300,6 +306,42 @@ def test_map_partitions():
     assert b.map_partitions(lambda a: len(a) + 1).name != b.map_partitions(len).name
 
 
+def test_map_partitions_with_kwargs():
+    corpus = '''
```

Comment:
If its cheap to run then I won't say no to a test.  However, we do need a simpler unit test that is easy for the maintainer to quickly understand at 4am when things break and need to be resolved quickly.